rules/windows/command_and_control_certutil_network_connection.toml

```
@@ -26,22 +26,32 @@ in order to take the next steps in a compromised environment.
 This rule looks for network events where `certutil.exe` contacts IP ranges other than the ones specified in
 [IANA IPv4 Special-Purpose Address Registry](https://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xhtml)
 
+> **Note**:
+> This Investigation Guide uses the Osquery Markdown Plugin which was introduced in Elastic 8.5, so users using older Elastic versions will see the raw syntax of the buttons.
```

```suggestion
> This investigation guide uses the Osquery Markdown Plugin introduced in Elastic stack version 8.5.0. Older Elastic stacks versions will see unrendered markdown in this guide.
```

Potential verbiage adjustment that could be used since this will be in production.